### PR TITLE
Add CUDA topic

### PR DESCRIPTION
**Pull request by benchislett**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

NVIDIA's CUDA C++ extension is widely used to develop GPU-capable code. I would like to browse projects using CUDA as a topic.

---------------------------------------------------------------------

### CODE REVIEW

File: topics/cuda/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+display_name: CUDA
+logo: cuda.png
+released: June 23, 2007
+short_description: CUDA is a parallel computing platform and programming model for NVIDIA GPUs.
+topic: cuda
+url: https://developer.nvidia.com/cuda-zone
+related: nvcc
+wikipedia_url: https://en.wikipedia.org/wiki/CUDA
+---
+CUDA® is a parallel computing platform and programming model developed by NVIDIA for general computing on graphical processing units (GPUs). With CUDA, developers are able to dramatically speed up computing applications by harnessing the power of GPUs.
```
Comment (benchislett) [inline topics/cuda/index.md:12]: Got the blurb from [here](https://developer.nvidia.com/cuda-zone). Do I need to do anything for licensing, or is it fine as-is?

**Review by deniseyu — APPROVED** [review]

File: topics/cuda/index.md (hunk 1 above)
Comment (deniseyu) [reply topics/cuda/index.md:12]: Should be OK as is! Thanks for the contribution!

**Comment by deniseyu** [comment]
@benchislett would you mind hitting the "Update branch" button - the tests should go green after that and I'll be happy to merge this.

**Comment by deniseyu** [comment]
@benchislett it looks like the image has to be 288px wide! as soon as that's resolved, this should auto-merge ✨

**Pull request merged**